packages/storybook/src/components-react/heading/Heading.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, Controls, Description, Markdown, Meta } from "@storybook/blocks";
+import markdown from "@utrecht/heading-css/README.md?raw";
+import * as HeadingStories from "./Heading.stories";
+import { CitationDocumentation } from "../../utils/CitationDocumentation";
+
+<Meta of={HeadingStories} />
+
+# Lux Heading
+
+Lux Heading is gebasseerd op Utrecht Heading.
+
+<CitationDocumentation
+  component="Utrecht Heading"
+  url="https://nl-design-system.github.io/utrecht/storybook-css/index.html?path=/docs/css-heading--docs"
+/>
+
+<Markdown>{markdown}</Markdown>
+
+## Utbreidingen in Lux Heading ten opzichte van Utrecht Heading
```

Comment:
```suggestion
## Uitbreidingen in Lux Heading ten opzichte van Utrecht Heading
```

Apart from that: this block doesn't add value as the user doens't need to know which component is being used underwater.

Reply:
In that sense we can remove the first item in the following list. The 3rd item is also not needed ([see other comment](packages/components-react/src/heading/Heading.tsx#r13)). The other two items should be named as they already are.